src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -839,12 +834,31 @@ private void AggregateRenderPackages(PackageAggregationParams parameters)
                         var pt = meshSet.Positions[idxCount];
                         parameters.Text.TextInfo.Add(new TextInfo(HelixRenderPackage.CleanTag(rp.Description), new Vector3(pt.X + 0.025f, pt.Y + 0.025f, pt.Z + 0.025f)));
                     }
-
-                    MeshCount++;
                 }
             }
         }
 
+        private void Detach(bool detachPersistentObjects)
```

Comment:
@ramramps I'm not sure how we detach in the new world (where we dynamically create Model3D objects). Detach is the correct way to have Helix dispose of unmanaged memory used by SharpDX/DirectX.